src/Utility/Log.cpp

```diff
@@ -113,6 +129,8 @@ Log& Log::operator<<(const glm::vec3& value) {
 
 Log& Log::operator<<(const glm::vec4& value) {
     *streams[currentChannel] << "(" << value.x << "," << value.y << "," << value.z << "," << value.w << ")";
+    if (currentChannel != INFO)
+        std::cout << "(" << value.x << "," << value.y << "," << value.z << "," << value.w << ")";
```

Comment:
This line is more or less a copy of the line above. Perhaps storing the output in a temporary string variable would be cleaner.